rollup.config.js

```diff
@@ -0,0 +1,29 @@
+import commonjs from 'rollup-plugin-commonjs';
+import builtins from 'rollup-plugin-node-builtins';
+
+export default {
+  input: 'index.js',
+  output: {
+    file: 'dist/index.umd.js',
+    format: 'umd',
+    name: 'devicePilot',
+    external: [
```

Comment:
This key needs to be in the root object (https://rollupjs.org/guide/en/#warning-treating-module-as-external-dependency) - we don't want build warnings.